controllers/admin/AdminCartRulesController.php

```diff
@@ -171,9 +171,7 @@ public function postProcess()
                     $rule_group_id = 1;
                     if (is_array($rule_group_array)) {
                         // Empty for (with a ; at the end), that just find the first rule_group_id available in rule_group_array
-                        for ($rule_group_id = 1; in_array($rule_group_id, $rule_group_array); ++$rule_group_id) {
-                            42;
-                        }
```

Comment:
@matks Now I look at the comment and think that the logic of the code is not correct. The comment states that it finds the first value available in the array.

Reply:
@PierreRambaud What do you think here?

Reply:
I agree with you, the statement is useless, but if you're thinking algorithm, it should be a whil it's in array, then increase the var